Edits to the EA handbook (#906)
kubernetes/sig-release: release-team/role-handbooks/emeritus-adviser/README.md

## Overview

> The Emeritus Adviser (EA) is an experienced Release Team (RT) member who returns to the team in an advisory role.  

Reviewer: ```suggestion
The Emeritus Adviser (EA) is an experienced Release Team (RT) member who returns to the team in an advisory role.
```

### Requirements and Selection

> The EA should be an individual who has served on the release team, in a lead capacity.  The primary selection pool should be former release leads (ideally from the previous 4 releases), however if there are no available individuals the EA can come from other former role leads who have been an exemplary member of the previous release teams.  

Reviewer: ```suggestion
The EA should be an individual who has served on the Release Team, in a Lead capacity.  The primary selection pool should be former Release Team Leads (ideally from the previous 4 releases). However, if there are no available individuals the EA can be another former Role Lead, who has been an exemplary member of a previous Release Team or a SIG Release subproject owner e.g., Release Engineering.
```

An ideal Emeritus Adviser is inherently suited to people management, has shown sustained interest in the health and growth of SIG Release, and has a commitment to Release Team diversity and helping others ascend the [Contributor Ladder](https://git.k8s.io/community/community-membership.md).

Time commitments for the EA are relatively light, with the largest time commitment in the two-week period where Shadows are being selected.  However, the EA role starts 2-3 weeks before the release cycle officially begins, and continues for 1-2 weeks after it ends.

> EAs are not chosen as part of the regular RT selection process.  Instead, interested candidates should contact the outgoing EA and the incoming Release Lead 2 weeks before the final prior release.  Most of the time, the outgoing EA will solicit suitable candidates.

Reviewer: We need to tweak this based on our discussion.
I think this can be an Emeritus Adviser's _suggestion_, but the _selection_ should be made by the outgoing and incoming leads (to make sure it's inclusive).

Author: just to clarify this is talking about the selection of the EA role (not the release team lead succession) 

though if that is unclear can put in more clear wording that this is for EA succession

Reviewer: Yeah, this needs a tweak as we've tapped people for this role in previous cycles.
I think the outgoing Emeritus Adviser should put forth a set of candidates which are then selected from the outgoing and incoming Release Team Leads.

Author: to clarify my understanding the suggestion is that hypothetically for this cycle I would have nominated 2-3 people and had Guin and Jorge select an individual to be EA for 1.18?

Reviewer: Yep!

Author: updated

The EA for the next release team should be selected ahead of opening up nominations for the next release cycle, and the incoming EA will be assigned that issue. 

Due to the nature of the role, the EA has no shadows.  The job may, however, be split between more than one person.

## Responsibilities

> Responsibilities for the EA is to help enable the release team and lead.  Activities the EA is responsible for are 

Reviewer: ```suggestion
Responsibilities for the EA are to help enable the Release Team. Activities the EA is responsible for are 
```

* Tracking the former release retrospective AIs 
* Creating the incoming release shadow surveys 
* Assisting release role leads in shadow selection 
* Running a general shadow orientation for folks, explaining the basics of the release team 
* Checking in on leads and shadows throughout the release and offering support to them 
* Conducting Shadow experience surveys throughout the release cycle and a lead/mentor questionaire at the end of the cycle

> * Helping facilitate the release retro (if no other facilitator can be found) 

Reviewer: I think we landed on wanting the EA to be the facilitator of first resort?
cc: @lachie83

Author: updated

EAs are not chosen as part of the regular RT selection process.  Instead, interested candidates should contact the outgoing EA and the incoming Release Lead 2 weeks before the final prior release.  Most of the time, the outgoing EA will solicit suitable candidates.

The EA for the next release team should be selected ahead of opening up nominations for the next release cycle, and the incoming EA will be assigned that issue. 

Due to the nature of the role, the EA has no shadows.  The job may, however, be split between more than one person.

## Responsibilities
Responsibilities for the EA are to help enable the Release Team. Activities the EA is responsible for are 

* Tracking the former release retrospective AIs 
* Creating the incoming release shadow surveys 
* Assisting release role leads in shadow selection 
* Running a general shadow orientation for folks, explaining the basics of the release team 
* Checking in on leads and shadows throughout the release and offering support to them 
* Conducting Shadow experience surveys throughout the release cycle and a lead/mentor questionaire at the end of the cycle

> * Helping facilitate the release retro (if no other facilitator can be found) 

Reviewer 2: Change the verb form at the beginning to Track, Create, Assist etc to be consistent with the rest of the handbook

Author: updated verb-age

2. Make any changes to it identified as required in the Retro or elsewhere.
3. When the new RT Lead gives the OK, open the form and announce it in the following places: #sig-release channel, sig-release mailing list, kubernetes-dev mailing list, and the Kubernetes Community Meeting.
4. Close the Shadow Application around 7-10 days after the initial announcement.
5. Chat with each of the incoming Role Leads about selecting their shadows, and make sure that they stay on schedule for it.
6. Once all role leads select their shadows the EA will send out a notification to all applicants not selected for this release cycle 

The most time-consuming part of this is helping the Role Leads select shadows.  In addition to the usual dilemmas of too many good candidates, the EA needs to give advice that makes sure that a diverse pool of shadows is selected, and that the Role Lead doesn't take on more shadows than they can effectively mentor.

Occasionally, a role may not get enough qualified candidates, in which case it's the EA's job to work with the new Role Lead to find or solicit additional candidates.

### Shadow and Lead Status 

Weeks: 1 to 12

> First, have a "shadow orientation" meeting sometime in week 2 or week 3, after all shadows are selected.  This meeting will likely have to be split for time zone reasons and held twice.  In this meeting, go over the information in the main Shadow docs, give information about the release team and SIG-Release, and then open things for questions or discussion. Make sure that Shadows know they can come to you if they have problems during their apprenticeship.

Reviewer: ```suggestion
First, have a "shadow orientation" meeting sometime in week 2 or week 3, after all shadows are selected.  This meeting will likely have to be split for time zone reasons and held twice.  In this meeting, go over the information in the main Shadow docs, give information about the release team and SIG Release, and then open things for questions or discussion. Make sure that Shadows know they can come to you if they have problems during their apprenticeship.
```

> For leads, ensure they have enough support for their role, and to be an effective mentor to their shadows.  

Reviewer: ```suggestion
**For leads, ensure they have enough support for their role, and to be an effective mentor to their shadows.**
```
